tests/perftest-plot.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+
+""" Plot the perftests. """
+
+import argparse
+import glob
+import os.path
+
+import matplotlib.pylab
+import numpy
+
+# Constants for libcperciva's perftests
+_COLUMN_NAMES = ["blocksizes", "speed MB/s"]
```

Comment:
As indicated, these column names are hard-coded for libcperciva's (current) perftests.  At the moment, I'm leaning towards using this script for the spiped perftests as well, but using a command-line option to override the column names when necessary.

Another option is to write a YAML file with metadata about the particular perftests, but at the moment that feels like a prematurely over-engineered solution.  Also, I could just have make a copy of this file for spiped, with minor difference for project-specific info (like we do for `Makefile.BSD`).